components/time-select/select-surface.tsx

```diff
@@ -17,6 +17,9 @@ import useMeasure from 'react-use-measure';
 import useSWR from 'swr';
 import useTranslation from 'next-translate/useTranslation';
 
+import ChevronLeft from 'components/icons/chevron-left';
```

Comment:
When you import icon components, you should make sure to add the "Icon" ending so other developers know that it's an icon. This should be:

```tsx
import ChevronLeftIcon from 'components/icons/chevron-left';
```